quickstarts/microsoft.digitaltwins/digitaltwins-with-function-private-link/main.bicep

```diff
@@ -0,0 +1,93 @@
+@description('The location into which the Azure Storage resources should be deployed.')
+@allowed([
+  'westcentralus'
+  'westus2'
+  'northeurope'
+  'australiaeast'
+  'westeurope'
+  'eastus'
+  'southcentralus'
+  'southeastasia'
+  'uksouth'
+  'eastus2'
+])
+param location string
+
+@description('Azure Digital Twins instance name')
+param digitalTwinsInstanceName string = 'digitaltwins-${uniqueString(resourceGroup().name)}'

```

Comment:
use ```resourceGroup().id``` as the seed for uniqueString (here and below) - the name is not globally unique

Reply:
Thanks, addressing.